### ISSUE TITLE
Auto Update not working

### ISSUE THREAD

**Issue by doctorspooky** [sepluginloader/SpaceEngineersLauncher#15]
Auto updates fail after I tell it to update.  I will manually download and install - maybe I won't get SE load-up crashes after that!

**Comment by austinvaness** [comment]
Can I see the launcher.log file in the Bin64/Plugins/ folder?

**Comment by bid** [comment]
Yep, something broke today. Yesterday evening all was still fine. I don't get crashes, but I just loaded into my current game and the planet I am on is much smaller than it should be, making all my structures float high in the air. I assume that the planet used to be bigger than 120km and the "Large Planets" plugin doesn't work anymore? Maybe?

Sadly, I didn't pay attention and can't tell if the launcher recompiled the plugins today. 

[launcher.log](https://github.com/sepluginloader/SpaceEngineersLauncher/files/11518951/launcher.log)
[loader.log](https://github.com/sepluginloader/SpaceEngineersLauncher/files/11518953/loader.log)

**Comment by doctorspooky** [comment]
[launcher.log](https://github.com/sepluginloader/SpaceEngineersLauncher/files/11519000/launcher.log)
[loader.log](https://github.com/sepluginloader/SpaceEngineersLauncher/files/11519001/loader.log)


There's an error listed in loader.log too, so I pasted that too (pretty much the same error.

I downloaded the current .exe, but still getting the "update to new version" message.

**Comment by doctorspooky** [comment]
> Yep, something broke today. Yesterday evening all was still fine. I don't get crashes, but I just loaded into my current game and the planet I am on is much smaller than it should be, making all my structures float high in the air. I assume that the planet used to be bigger than 120km and the "Large Planets" plugin doesn't work anymore? Maybe?
> 
> Sadly, I didn't pay attention and can't tell if the launcher recompiled the plugins today.
> 
> [launcher.log](https://github.com/sepluginloader/SpaceEngineersLauncher/files/11518951/launcher.log) [loader.log](https://github.com/sepluginloader/SpaceEngineersLauncher/files/11518953/loader.log)

The Camera to LCD plugin is the only one listed as returning errors in my plugin list (maybe they have to be reported before PluginLoader lists them so?).  Possibly an SE issue?

**Comment by austinvaness** [comment]
The errors suggest that something is interfering with DNS on your network. Can you go to the urls from the log files, https://github.com/sepluginloader/PluginHub/archive/main.zip and https://github.com/sepluginloader/PluginLoader/releases/download/v1.11.3/PluginLoader-v1.11.3.zip?

**Comment by doctorspooky** [comment]
> The errors suggest that something is interfering with DNS on your network. Can you go to the urls from the log files, https://github.com/sepluginloader/PluginHub/archive/main.zip and https://github.com/sepluginloader/PluginLoader/releases/download/v1.11.3/PluginLoader-v1.11.3.zip?

Yes.

**Comment by austinvaness** [comment]
Do you have an antivirus? It might be blocking plugin loader

**Comment by graywizardx** [comment]
I am having a very similar issue.

I dont see errors in the logs, and the URLs download fine when tried manually.

[launcher.log](https://github.com/sepluginloader/SpaceEngineersLauncher/files/14533258/launcher.log)
[loader.log](https://github.com/sepluginloader/SpaceEngineersLauncher/files/14533259/loader.log)

Game Version: 1.203.630

IPv6 is enabled
No AV or firewall rules
I have github working in other places (personal development)

Launcher is unblocked and in: D:\Program Files (x86)\Steam\steamapps\common\SpaceEngineers\Bin64 next to SpaceEngineers.exe

Any help would be greatly appreciated.
